src/components/MySongsView/MySongsViewTable/MySongsViewTable.html

```diff
@@ -0,0 +1,20 @@
+<table class="mySongsTable">
+  <caption class="mdc-typography--headline4 mySongsTable__caption">My Songs</caption>
+  <thead>
+    <tr>
+      <th class="mySongsTable__th"></th>
+      <th class="mySongsTable__th"></th>
+      <th class="mySongsTable__th mySongsTable__th_name ">Name</th>
+      <th class="mySongsTable__th mySongsTable__th_duration">
+        <i class="material-icons">watch_later</i>
+      </th>
+      <th class="mySongsTable__th mySongsTable__th_artist">Artist</th>
+      <th class="mySongsTable__th mySongsTable__th_album">Album</th>
+      <th class="mySongsTable__th mySongsTable__th_3dots">3dot</th>
+    </tr>
+  </thead>
+  <tbody class="mySongsTable__body">
+    <tr class="mySongsTable__row mySongsTable__row1"></tr>
```

Comment:
how are you supposed to display several rows? please use loops or foreach and display data from array